spec/models/finance/schedule/ecf_spec.rb

```diff
@@ -32,7 +32,7 @@
     end
   end
 
-  Cohort.where.not(start_year: 2020).find_each do |cohort|
+  Cohort.where(start_year: 2021..Cohort.current.start_year).find_each do |cohort|
```

Comment:
So I understand why this passes now on first run (like in CI) but fails on the second run: on first run because we don't have all the cohorts it doesn't run these tests, but on second one after it set up everything it does run and fails because of what you said 👍 let's keep it in, thank for fixing